examples/basic-datasource/provisioning/dashboards/default.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: 1
+
+providers:
+  - name: Default
+    type: file
+    options:
+      path: /etc/grafana/provisioning/dashboards
```

Comment:
There doesn't seem to be a dashboard.json to accompany this example. Should we add one?